<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Eksportuj</class>
 <widget class="QWidget" name="Eksportuj">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Biblioteka Filmów - Kreator Eksportu</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_Eks_path">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>18</y>
     <width>310</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_GetPath">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>18</y>
     <width>49</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>3</y>
     <width>307</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Podaj ścieżkę do pliku, do którego chesz wyekportować dane</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_AllRec">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>51</y>
     <width>265</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Eksportuj wszystkie rekordy filmow z bazy danych</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>75</y>
     <width>352</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Wybierz jakie dane mają być eksportowane</string>
   </property>
   <widget class="QCheckBox" name="checkBox_MainFilm">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>21</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Podstawowe dane o filmie</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_TabObsada">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>45</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tabela Obsada</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_TabOcena">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>69</y>
      <width>94</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tabela Ocena</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_TabLZ">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>93</y>
      <width>163</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tabela Lokalizacje zdjęciowe</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_TabPD">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>117</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tabele Produkcja i Dystrybucja</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_TabW">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>141</y>
      <width>124</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tabele Wypożyczeń</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Eksport">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>273</y>
     <width>106</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Eksportuj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Anuluj">
   <property name="geometry">
    <rect>
     <x>291</x>
     <y>273</y>
     <width>75</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Anuluj</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
